test/e2e/specs/editor/various/preferences.spec.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'preferences', () => {
+	test( 'remembers sidebar dismissal between sessions', async ( {
+		page,
+		admin,
+	} ) => {
+		await admin.createNewPost();
+		// Open by default.
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Post' );
+
+		// Change to "Block" tab.
+		await page.click( 'role=button[name="Block"i]' );
+
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Block' );
+
+		// Regression test: Reload resets to document tab.
+		//
+		// See: https://github.com/WordPress/gutenberg/issues/6377
+		// See: https://github.com/WordPress/gutenberg/pull/8995
+		await page.reload();
+		await page.waitForSelector( '.edit-post-layout' );
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Post' );
+
+		// Dismiss.
+		await page.click(
+			'.edit-post-sidebar__panel-tabs [aria-label="Close settings"]'
+		);
+		await expect(
+			page.locator( '.edit-post-sidebar__panel-tab.is-active' )
+		).not.toBeVisible();
+
+		// Remember after reload.
+		await page.reload();
+		await page.waitForSelector( '.edit-post-layout' );
```

Comment:
I'm not a huge fan of this line because it waits for something that's unrelated to the user interface being tested.

If it could be changed to a role selector for this element, that would be great:
![Screen Shot 2022-09-15 at 5 20 37 pm](https://user-images.githubusercontent.com/677833/190366888-47857122-f1ea-4632-bd69-9f8bf59ef8cb.png)